cmd/du.go

```diff
@@ -103,7 +103,12 @@ func newDuCmd() *cobra.Command {
 
 func (o *duOptions) run(cmd *cobra.Command) error {
 	filterNames := []string{"size", "name"}
-	_, searchFilter, err := utils.ParseFilterParameters(o.filter, filterNames)
+	filters := utils.NewFilters(nil, filterNames)
```

Comment:
Minor: maybe `NewFilters()` could also accept the filter values (`o.filter`) and call `AddFilters()` internally to initialize?
Or do you think there is a reason to keep the two steps separated?

I mean it could be used like this:
```go
filters, err := utils.NewFilters(nil, filterNames, o.filter)
if err != nil {
    return err
}
```

Reply:
I agree, it becomes less verbose this way